packages/cli-lib/tests/unit/fixtures/comp.gts

```diff
@@ -0,0 +1,69 @@
+import formatMessage from 'ember-intl/helpers/format-message';
+import or from 'ember-truth-helpers/helpers/or';
+import Component from '@glimmer/component';
+import {service} from '@ember/service';
+
```

Comment:
I think if a second component were added to this file, the implementation (as is) would break -- unless I misunderstanding something about how `ember-template-tag` works opposed to `content-tag`